src/widget/form/chatform.cpp

```diff
@@ -75,21 +75,7 @@ namespace
     QString statusToString(const Status status)
     {
         QString result;
-        switch (status) {
-        case Status::Online:
-            result = ChatForm::tr("online", "contact status");
-            break;
-        case Status::Away:
-            result = ChatForm::tr("away", "contact status");
-            break;
-        case Status::Busy:
-            result = ChatForm::tr("busy", "contact status");
-            break;
-        case Status::Offline:
-            result = ChatForm::tr("offline", "contact status");
-            break;
-        }
-        return result;
+        return ChatForm::tr(Widget::getStatusTitle(status).toLatin1().data(), "contact status");
```

Comment:
1. I hope you plan to move `getStatusTitle` outside of `Widget`. Otherwise you make `ChatForm` depends on Widget

Reply:
I know, that I'm writing comment to the already merged PR, but this part lead to real problems with translation. Just run `./tools/update-translation-files.sh ALL` and check diff. Strings with "away", "busy", etc. are removed.